tests/system/libraries/VSCodeLib.py

```diff
@@ -0,0 +1,184 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2025 Bill Dengler
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+import os as _os
+import re as _re
+import shutil as _shutil
+import tempfile as _tempfile
+import json as _json
+from robot.libraries.BuiltIn import BuiltIn as _BuiltIn
+from SystemTestSpy import _blockUntilConditionMet, _getLib
+from SystemTestSpy.windows import (
+	CloseWindow as _CloseWindow,
+	GetWindowWithTitle as _GetWindowWithTitle,
+	SetForegroundWindow as _SetForegroundWindow,
+	Window as _Window,
+	windowWithHandleExists as _windowWithHandleExists,
+)
+import NvdaLib as _NvdaLib
+
+from robot.libraries.Process import Process as _ProcessLib
+import WindowsLib as _WindowsLib
+
+_builtIn: _BuiltIn = _BuiltIn()
+_process: _ProcessLib = _getLib("Process")
+_windowsLib: _WindowsLib = _getLib("WindowsLib")
+
+_LAUNCHER_PATHS = [
+	_os.path.expandvars(r"%LOCALAPPDATA%\Programs\Microsoft VS Code\Code.exe"),
+	_os.path.expandvars(r"%ProgramFiles%\Microsoft VS Code\Code.exe"),
```

Comment:
this expands to `Program Files (x86)` on 32bit interpreters

```suggestion
	_os.path.expandvars(r"C:\Program Files\Microsoft VS Code\Code.exe"),
```